public/v4/apps/minio.yml

```diff
@@ -14,7 +14,7 @@ services:
         caproverExtra:
             containerHttpPort: '9001'
             dockerfileLines:
-                - FROM minio/minio:RELEASE.2021-08-17T20-53-08Z
+                - FROM minio/minio:$$cap_minio_version
```

Comment:
Thanks for the PR!

I'll change the default version to `RELEASE.2021-08-17T20-53-08Z` after merging since that is the tested version.